build/scripts/build.ps1

```diff
@@ -27,7 +27,6 @@ param (
     [switch]$sign = $false,
     [switch]$pack = $false,
     [switch]$packAll = $false,
-    [switch]$binaryLog = $false,
```

Comment:
Why are you deleting this option? There is a perf difference when building with / without this option. This is why it wsa left an option.

Reply:
Okay

Reply:
You are right. Binary log slows down Roslyn build by 1 minute (4:15 vs 3:15). I'll make it opt-in again.